test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -88,6 +89,8 @@ public static RunLoadTest defaultLoadTest(Supplier<HapiSpecOperation[]> opSource
 				.tolerance(settings::getTolerancePercentage)
 				.allowedSecsBelow(settings::getAllowedSecsBelow)
 				.setNumberOfThreads(threadNumber.isPresent() ? threadNumber::getAsInt : settings::getThreads)
+				.setHCSSubmitMessageSize(
+						hcsSubmitMessage.isPresent() ? hcsSubmitMessage::getAsInt : settings::getHcsSubmitMessageSize)
```

Comment:
if hcsSubmitMessage has no valid value, then use settigns. But hcsSubmitMessage seems never being assigned in any code. Maybe it should be assigned in SubmitMessageLoadTest.java  when cmd line parameter is given to SubmitMessageLoadTest.java

Reply:
@JeffreyDallas  Thanks ! Assigned the 4th parameter as `hcsSubmitMessageSize`